src/components/dialog/dialog.scss

```diff
@@ -7,7 +7,7 @@ $dialog-padding: $baseline-grid * 3;
   display: flex;
   justify-content: center;
   align-items: center;
-  position: fixed;
+  position: absolute;
```

Comment:
I'd like to hear from @rschmukler on this one, just to confirm that this is safe and we aren't missing some edge case.